docs/articles/documentation/test-api/actions/take-screenshot.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+title: Take Screenshot
+permalink: /documentation/test-api/actions/take-screenshot.html
+---
+# Take Screenshot
+
+Takes a screenshot of the tested page.
+
+```text
+takeScreenshot( [path] )
+```
+
+Parameter           | Type   | Description                                                                                           | Default
+------------------- | ------ | ----------------------------------------------------------------------------------------------------- | ----------
+`path` *(optional)* | String | A relative path to the folder where screenshots should be saved. Resolved from the *screenshot directory* specified via an [API](../../using-testcafe/programming-interface/runner.md#screenshots) or [CLI](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) option. | The screenshot directory specified via an [API](../../using-testcafe/programming-interface/runner.md#screenshots) or [command-line interface](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) option.
+
+> Important! If the screenshot directory is not specified in [API](../../using-testcafe/programming-interface/runner.md#screenshots) or [command-line interface](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path),
+> the `takeScreenshot` action is ignored.
+
+The following example shows how to use the `takeScreenshot` action.
+
+```js
+test('Take a screenshot of my new avatar', async t => {
+    await t
+        .click('#change-avatar')
+        .setFilesToUpload('#upload-input', 'img/portrait.jpg')
```

Comment:
I think we can get rid of native dialogs handling in this example to simplify it. What do you think?